[2024-03-05T09:00:16.902+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: defaultargs.tsk3 scheduled__2024-03-05T08:00:00+00:00 [queued]>
[2024-03-05T09:00:16.939+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: defaultargs.tsk3 scheduled__2024-03-05T08:00:00+00:00 [queued]>
[2024-03-05T09:00:16.940+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-05T09:00:16.940+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-03-05T09:00:16.941+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-05T09:00:16.990+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk3> on 2024-03-05 08:00:00+00:00
[2024-03-05T09:00:17.001+0000] {standard_task_runner.py:55} INFO - Started process 74 to run task
[2024-03-05T09:00:17.007+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'defaultargs', 'tsk3', 'scheduled__2024-03-05T08:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/defaultargs.py', '--cfg-path', '/tmp/tmp8o334r5r']
[2024-03-05T09:00:17.008+0000] {standard_task_runner.py:83} INFO - Job 285: Subtask tsk3
[2024-03-05T09:00:17.036+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-03-05T09:00:17.096+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/sqlalchemy.py:124: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  return not conf.get("database", "sql_alchemy_conn").startswith("mssql")

[2024-03-05T09:00:17.208+0000] {task_command.py:388} INFO - Running <TaskInstance: defaultargs.tsk3 scheduled__2024-03-05T08:00:00+00:00 [running]> on host df7038a2f7dd
[2024-03-05T09:00:17.485+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=jezandre_tiago@hotmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=defaultargs
AIRFLOW_CTX_TASK_ID=tsk3
AIRFLOW_CTX_EXECUTION_DATE=2024-03-05T08:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-05T08:00:00+00:00
[2024-03-05T09:00:17.487+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-03-05T09:00:17.489+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 5']
[2024-03-05T09:00:17.504+0000] {subprocess.py:86} INFO - Output:
[2024-03-05T09:00:22.513+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-05T09:00:22.864+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=defaultargs, task_id=tsk3, execution_date=20240305T080000, start_date=20240305T090016, end_date=20240305T090022
[2024-03-05T09:00:23.062+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-05T09:00:23.094+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
